### PR TITLE
Feature/top bar 

### PR DESCRIPTION
**Pull request by RachitMalik12**
This PR introduces a wrapper for the TopBar that has controls for user accounts. 
**Important to note:** Regarding the position of the TopBar being to the right of the sidebar or being above the sidebar.  One argument for having the top bar to the right of the sidebar is that it will have controls related to the view in focus (except the accounts) in the MainContent. If we have the TopBar above the sidebar we should probably move the Logo to the TopBar and make some more changes. I feel we should have a discussion on this and get everyone aligned. For now because of @qgolsteyn work with the orchestration view I have left it the way it is so we can be consistent with his work and so we have a consistent layout throughout.

**Approach**
The component TopBarAccountsWrapper takes 2 props: navigation and actions. Navigation is could be a Breadcrumb describing the current page and actions is a React Element that can be an overflow menu which will be placed as additional controls related to the view in focus. 

**Testing instructions**
I have implemented it in the Dashboard view for now with no actions.  So you should be able to access your account info and logout on clicking the little avatar.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "editor.formatOnSave": true
```
Comment (qgolsteyn) [inline .vscode/settings.json]: Whoops, maybe remove that?

File: frog/imports/containers/TopBarWrapper/types.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+export type TopBarWrapperPropsT = {
```
Comment (qgolsteyn) [inline frog/imports/containers/TopBarWrapper/types.js]: Move this out of types and into TopBarAccountsWrapper

File: frog/imports/client/UserDashboard/components/DashboardSideBar/index.js [hunk 3]
```diff
@@ -5,7 +5,7 @@ import { Sidebar, Panel } from '/imports/ui/Sidebar';
 import { Logo } from '/imports/ui/Logo';
 import { SidebarLayout } from '/imports/ui/Layout/SidebarLayout';
 import { RowButton, RowTitle } from '/imports/ui/RowItems';
-import { TopBar } from '/imports/ui/TopBar';
+import { TopBarAccountsWrapper } from '/imports/containers/TopBarWrapper/TopBarAccountsWrapper';
```
Comment (qgolsteyn) [inline frog/imports/client/UserDashboard/components/DashboardSideBar/index.js]: Rename TopBarAccountWrapper to index.js

File: frog/imports/ui/RowItems/rowButton.stories.js [hunk 4]
```diff
@@ -10,7 +10,7 @@ import { StorybookContainer } from '/imports/ui/StorybookContainer';
 storiesOf('UI/RowItems/RowButton', module)
   .add('with text only', () => (
     <StorybookContainer width={250}>
-      <RowButton>Hello World!</RowButton>
+      <RowButton> Hello World!</RowButton>
```
Comment (qgolsteyn) [inline frog/imports/ui/RowItems/rowButton.stories.js]: NIT: Remove single space

File: frog/imports/containers/TopBarWrapper/TopBarAccountsWrapper.js [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+                </RowButton>
+
+                <RowButton onClick={doLogout} icon={<LockOutlinedIcon />}>
+                  Logout
+                </RowButton>
+              </React.Fragment>
+            )}
+          </OverflowMenu>
+
+          <React.Fragment>{actions}</React.Fragment>
```
Comment (qgolsteyn) [inline frog/imports/containers/TopBarWrapper/TopBarAccountsWrapper.js]: NIT: `React.Fragment` can be changed to `<>`

File: frog/imports/containers/TopBarWrapper/TopBarAccountsWrapper.js [hunk 6]
```diff
@@ -0,0 +1,101 @@
+// @flow
+import * as React from 'react';
```
Comment (qgolsteyn) [inline frog/imports/containers/TopBarWrapper/TopBarAccountsWrapper.js]: NIT: Organize the import statements

**Review by qgolsteyn — CHANGES_REQUESTED** [review]
Some stylistic comments, but the functionality looks good!

**Review by houshuang — DISMISSED** [review]
Resolve the tiny changes and I'll merge.

**Pull request merged**